docs/src/test-advanced-js.md

```diff
@@ -120,7 +120,7 @@ In addition to everything from the [`workerInfo`](#workerinfo), the following in
 - `timeout: number` - Test timeout.
 - `annotations` - [Annotations](./test-annotations.md) that were added to the test.
 - `snapshotSuffix: string` - Suffix used to locate snapshots for the test.
-- `snapshotPath(snapshotName: string)` - Function that returns the full path to a particular snapshot for the test.
+- `snapshotPath(pathSegments: string[])` - Function that returns the full path to a particular snapshot for the test.
```

Comment:
> I changed this and array in 311fa94 but lmk if you actually wanted the spread.

Sorry, I was confused by some lint error. Spread is backwards compatible, so spread for sure! Luckily (or actually not!) there are still some tests failing, so this is not the only change to be made 😕

Reply:
No worries, simple fix. See 76efbc2.